Workbooks/EventHub/AzureMonitor/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Overview",
+    "author": "Microsoft",
+    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 100 }, { "type": "eventhub-insights", "resourceType": "Azure Monitor", "order": 100 }]
```

Comment:
Just chatted with Anurag, I'll need to add these types (eventhub-insights, servicebus-insights, rediscache-insights) to the InsightsPortal first

Reply:
Just merged the PR with those types to InsightsPortal